src/test/java/org/mule/extension/sftp/SftpNegativeConnectivityTestCase.java

```diff
@@ -35,6 +35,7 @@
 
 @Feature(SFTP_EXTENSION)
 @Story("Negative Connectivity Testing")
+@Ignore
```

Comment:
Because I have seen in commits that you ignored the tests in the class and as a result, Alejandro Ianucci committed the Ignore annotation to the class because all of its tests are ignored.